### PR TITLE
Solves Issue #558

### PR DESCRIPTION
**Pull request by arunavkonwar**

### CODE REVIEW

**Comment by arunavkonwar** [comment]
@amritpurshotam the build fails for some reason, not sure what's the issue. I had to edit the `api/migrations/versions/f8ab033698a2_.py` file to set a default value to the column `nominations`. Works fine in local though!

File: api/app/utils/testing.py [hunk 1]
```diff
@@ -120,4 +121,14 @@ def add_to_db(self, obj):
     def tearDown(self):
         db.session.remove()
         db.reflect()
-        db.drop_all()
\ No newline at end of file
+        db.drop_all()
+
+    def create_application_form(self,
+                            is_open,
```
Comment (avishkar58) [inline api/app/utils/testing.py]: Please set defaults

File: api/app/utils/testing.py [hunk 2]
```diff
@@ -120,4 +121,14 @@ def add_to_db(self, obj):
...
+                            deadline,
+                            nominations):
+                            
+        application_form = ApplicationForm(is_open, deadline, nominations)
```
Comment (avishkar58) [inline api/app/utils/testing.py]: This doesn't look right?

File: api/app/applicationModel/models.py [hunk 3]
```diff
@@ -9,6 +9,7 @@ class ApplicationForm(db.Model):
     is_open = db.Column(db.Boolean(), nullable=False)
 
     event = db.relationship('Event', foreign_keys=[event_id])
+    nominations = db.Column(db.Boolean(), nullable=False)
 
     def __init__(self, event_id, is_open):
```
Comment (avishkar58) [inline api/app/applicationModel/models.py]: Please add nominations to the constructor

**Comment by avishkar58** [comment]
Hey @arunavkonwar - the tests are failing because there are a lot of other tests besides the applicationModel one that create application forms and aren't passing the new nominations flag to the constructor. So you'll have to search for those places in the various tests and update them to use your new `create_application_form` method - a little tedious, which is why we're trying to add these helper methods now to prevent having to do it next time we add a field :D

**Comment by arunavkonwar** [comment]
@avishkar58 this solved it. Thanks.

File: api/app/responses/tests.py [hunk 4]
```diff
@@ -76,7 +76,7 @@ def _seed_data(self):
             datetime.now(), datetime.now(), datetime.now(), datetime.now())
         _add_object_to_db(self.test_event)
         self.test_form = ApplicationForm(
-            self.test_event.id, True)
+            self.test_event.id, True, False)
```
Comment (avishkar58) [inline api/app/responses/tests.py]: Please change these to all your new method instead: `self.create_application_form(...)` - that way, next time someone makes a change to ApplicationForm, they don't have to update all these tests again...
(same comment applies to the other tests)

**Comment by arunavkonwar** [comment]
@amritpurshotam @avishkar58 can you take a look at this PR. Wondering if there needs any more changes. Thanks :)

**Pull request merged**